docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '2'
+services:
+  app:
+    build: .
+    volumes:
+    - .:/usr/src/app
+    - node-modules:/usr/src/app/node_modules/
```

Comment:
I'm guess this makes sure node_modules is cached, or loaded from existing system? That sounds cool.